microsite/data/plugins/aws-app-development.yaml

```diff
@@ -1,10 +1,10 @@
 ---
-title: AWS App Developer Tools
+title: OPA on AWS
 author: Amazon Web Services
 authorUrl: https://aws.amazon.com/
 category: Infrastructure
-description: Create and manage AWS Apps within Backstage
-documentation: https://github.com/awslabs/app-development-for-backstage-io-on-aws#readme
-iconUrl: https://github.com/awslabs/app-development-for-backstage-io-on-aws/blob/main/docs/images/AWS_logo.png?raw=true
+description: Orchestrate Platforms and Applications (OPA) allows customers to build and manage AWS Apps & Environments within Backstage
+documentation: https://opaonaws.io
+iconUrl: https://github.com/awslabs/app-development-for-backstage-io-on-aws/blob/main/website/static/img/white_OPA_text02.png?raw=true
```

Comment:
This logo might be a bit hard to read, it's white test on a grey background. Would this one be a bit better? https://raw.githubusercontent.com/awslabs/app-development-for-backstage-io-on-aws/main/docs/images/opa-logo.png

Reply:
The logo we chose has a transparent background and simple white text - we thought this would look good since the backstage website theme is dark color. The blue logo has also a transparent background and won't look good on a dark background.

Reply:
Oh, ok, that's great, guess my browser isn't making it clear that the background is transparent 👍